src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -681,8 +681,12 @@ private MenuBar createMenu() {
                         factory.createMenuItem(StandardActions.EXPORT_SELECTED, new ExportCommand(this, true, Globals.prefs)),
                         factory.createMenuItem(StandardActions.SAVE_SELECTED_AS_PLAIN_BIBTEX, new OldDatabaseCommandWrapper(Actions.SAVE_SELECTED_AS_PLAIN, this, stateManager))),
 
-                factory.createMenuItem(StandardActions.CONNECT_TO_SHARED_DB, new ConnectToSharedDatabaseCommand(this)),
-                factory.createMenuItem(StandardActions.PULL_CHANGES_FROM_SHARED_DB, new OldDatabaseCommandWrapper(Actions.PULL_CHANGES_FROM_SHARED_DATABASE, this, stateManager)),
+                new SeparatorMenuItem(),
+
+                factory.createSubMenu(StandardActions.REMOTE_DB,
```

Comment:
Should this maybe be "shared database" for consistency? Or rename "shared" to "remote" globally which might be a better name anyway.